build.cmd

```diff
@@ -1 +1,2 @@
-@dotnet run --project "%~dp0\tools\FakeItEasy.Build\FakeItEasy.Build.csproj" -- %*
+@cd %~dp0

```

Comment:
Relative paths are used inside `Program.cs`. If you don't change working directory, then the targets will fail when run from a subdirectory, e.g. `../build.cmd`.

Reply:
@blairconrad it only changes it for the scope of the script:

![image](https://user-images.githubusercontent.com/677704/44944649-be2e9600-adda-11e8-9744-374946a9a757.png)

Reply:
> Or I might just swap build.cmd for build.ps1 to force people to use PowerShell. 😉

I'd consider doing that for FakeItEasy too... I never use cmd anymore.